src/hpx/kokkos/config.hpp

```diff
@@ -0,0 +1,16 @@
+///////////////////////////////////////////////////////////////////////////////
+//  Copyright (c) 2019-2020 ETH Zurich
+//
+//  Distributed under the Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+///////////////////////////////////////////////////////////////////////////////
+
+#pragma once
+
+#include <hpx/config/version.hpp>
+
+#if HPX_VERSION_FULL >= 0x011100
```

Comment:
Isn't this >= 1.17.0, since it's a hexadecimal literal?
```suggestion
#if HPX_VERSION_FULL >= 0x010B00
```

Reply:
I think it's correct the way I proposed it, see https://github.com/STEllAR-GROUP/hpx/blob/f711e4192411cc049889ba85614f211d9e86538b/libs/core/config/cmake/templates/config_version.hpp.in#L28-L35

Also, yes, please do a patch release as you see fit.

Reply:
Oh... yes, that's interesting. And you're right, sorry for the noise.